client/settings/settings-section.scss

```diff
@@ -11,7 +15,7 @@
 		margin-bottom: 12px;
 
 		@include breakpoint( '>800px' ) {
-			flex: 0 1 328px;
+			flex: 0 0 25%;
```

Comment:
Otherwise this is how it would look like at some breakpoints
![Screen Shot 2021-04-26 at 5 32 33 PM](https://user-images.githubusercontent.com/273592/116159203-6e9ce700-a6b5-11eb-8cb4-4a3161c4b21a.png)